google/pubsub/v1/pubsub.proto

```diff
@@ -30,6 +30,7 @@ option java_multiple_files = true;
 option java_outer_classname = "PubsubProto";
 option java_package = "com.google.pubsub.v1";
 option php_namespace = "Google\\Cloud\\PubSub\\V1";
+option ruby_namespace = "Google::Cloud::PubSub::V1";
```

Comment:
[drive-by question]
What is the rationale for including the `ruby_namespace` now, if the v1 proto library has already been generated and in existence for a while?

Reply:
Mostly consistency with other languages that already use "PubSub" instead of "Pubsub". We are trying to get the `google-cloud-pubsub` gem to GA, so if we are going to make this change it should happen before then.

Reply:
I see.